tsl/src/remote/dist_copy.c

```diff
@@ -54,13 +60,36 @@ typedef struct CopyDimensionInfo
 	int32 atttypmod;
 } CopyDimensionInfo;
 
+typedef struct DataNodeConnection
+{
+	TSConnectionId id;
+	TSConnection *connection;
```

Comment:
Curious why we don't we have `TSConnectionId` field somewhere inside the `TSConnection` object, maybe it would make more sense then creating a new connection object?

Reply:
Probably... I'm just trying to avoid large refactorings, this PR is big enough already.